src/python/dxpy/bindings/dxapplet.py

```diff
@@ -184,9 +191,11 @@ def _get_cleanup_keys(self):
         raise NotImplementedError('_get_cleanup_keys is not implemented')
 
     def run(self, executable_input, project=None, folder=None, name=None, tags=None, properties=None, details=None,
```

Comment:
Any new arguments should be appended to the end of the arguments. This is a breaking change if the caller of the function is not using keyword arguments.